src/Data/provideLocalStorageDataClass.ts

```diff
@@ -25,6 +27,7 @@ export function provideLocalStorageDataClass(
   } = {},
 ) {
   const recordKey = `localDataClass-${scrivitoTenantId()}-${className}`
+  recordKeys.add({ className, recordKey })
```

Comment:
* [ ] FYI The `recordKey` can be recomputed easily from the `className`. How about reducing redundancy and just storing the list of class names?
* [ ] FYI Taking it one step further: Looking at the intended usage the caller of `searchLocalStorageDataClasses` [knows](https://github.com/Scrivito/scrivito-portal-app/commit/b3501f232fb4b4ead42ada2787f8376f59dbdf99#diff-f6b709a5b9754aed0e22f78271f8af54bb316ee2f4a4be41e2106558e39398b4R12-R20) all class names. If this is true for all callers we could  rid of the list of keys completely by taking the list of class names as an argument.